src/device/usbd_control.c

```diff
@@ -140,6 +140,14 @@ bool tud_control_xfer(uint8_t rhport, tusb_control_request_t const * request, vo
 // USBD API
 //--------------------------------------------------------------------+
 
+//--------------------------------------------------------------------+
+// Prototypes
+//--------------------------------------------------------------------+
+void usbd_control_reset(void);
```

Comment:
Does IAR complains without these prototypes, these are only used by `usbd.c`, typically they would have its own private header, though I try to cut-corner and put those prototypes inside the usbd.c diretly
https://github.com/hathach/tinyusb/blob/master/src/device/usbd.c#L264 .

Reply:
If the option `Require Prototypes` is enabled, IAR emit error about this.

Reply:
thanks for clarification, I guess there is no harm to have extra prototypes here.